test/tasks/maintenance/backfill_api_key_owners_task_test.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class Maintenance::BackfillApiKeyOwnersTaskTest < ActiveSupport::TestCase
+  test "#process performs a task iteration" do
+    element = create(:api_key)
+    element.update!(owner: nil)
```

Comment:
Will this not run validations and add the user back right here? Maybe add an assertion that owner is nil or use the update_columns call in the test below.